core/geometry/src/clipping/ConvexClipPlaneSet.ts

```diff
@@ -163,7 +163,7 @@ export class ConvexClipPlaneSet implements Clipper, PolygonClipper {
    * @param interior array whose boolean value is used as both the `interior` and `invisible` bits of the plane for the succeeding segment.

```

Comment:
Those bits default to `false` if `undefined` is supplied for this param, correct?

Reply:
correct.   comment expanded now.  tx